src/components/dev-hub/events.js

```diff
@@ -102,18 +103,18 @@ const DateIcon = ({ date }) => {
 };
 
 const Event = ({ event, maxTitleLines = 2, ...props }) => {
-    const [locationColor, setLocationColor] = useState(colorMap.greyLightThree);
+    const theme = useTheme();
```

Comment:
Let's check this component to make sure the hover states are working properly. Here it is on this branch:

![Screen Shot 2020-06-02 at 10 59 05 AM](https://user-images.githubusercontent.com/9064401/83535645-351d2500-a4c0-11ea-800c-1e66f41aecc8.png)

and here it is on prod:

![Screen Shot 2020-06-02 at 10 59 11 AM](https://user-images.githubusercontent.com/9064401/83535667-3cdcc980-a4c0-11ea-8a55-f4d369b6b63e.png)

Reply:
I still think something is missing here. On `hover` we should get the filled in event date like above but otherwise (and what this PR has now is correct) we should have the below with the gradient border. Check out `/community` on prod to see it!